### ISSUE TITLE
mysql-connector-python==2.1.4 does not exist in PIP repos (cant install)

### ISSUE THREAD

**Issue by BarbzYHOOL** [ShahriyarR/MySQL-AutoXtraBackup#368]
Error:

```
    :stderr:   Could not find a version that satisfies the requirement mysql-connector-python==2.1.4 (from mysql-autoxtrabackup==1.5.2) (from versions: 8.0.5, 8.0.6, 8.0.11, 8.0.12, 8.0.13, 8.0.14, 8.0.15, 8.0.16, 8.0.17, 8.0.18)
    No matching distribution found for mysql-connector-python==2.1.4 (from mysql-autoxtrabackup==1.5.2)

```

How can we make it work so that it downloads always the right dependency version?

There is this in the code:

```
    install_requires=[
        'click>=3.3',
        'mysql-connector-python==2.1.4',
        'pid>=2.0',
        'humanfriendly>=2.0',
        'pytest'
    ],
    dependency_links=['https://dev.mysql.com/get/Downloads/Connector-Python/mysql-connector-python-2.1.4.tar.gz'],
```

With the correct link, but apparently PIP tries to install it from PIP repos where it has been removed from.

@dakine1111 @ShahriyarR

**Comment by BarbzYHOOL** [comment]
ok what about this:

```
    install_requires=[
        'click>=3.3',
        'mysql-connector==2.2.9',
        'pid>=2.0',
        'humanfriendly>=2.0',
        'pytest'
    ],
```

**Comment by BarbzYHOOL** [comment]
I did that, it installed correctly but I haven't tried the program itself yet

**Comment by ShahriyarR** [comment]
Fixed as part of #347
